packages/shared/src/tokens/evm/config.ts

```diff
@@ -72,6 +72,18 @@ export const evmTokenConfig: Record<EvmNetworks, Partial<Record<EvmToken, EvmTok
       type: TokenType.Evm
     }
   },
+  [Networks.PolygonAmoy]: {
+    [EvmToken.USDC]: {
```

Comment:
Let's remove this token since it's not really usable for a transfer as far as I understand.